client/rpcserver/rpcserver.go

```diff
@@ -46,13 +51,14 @@ const (
 var (
 	// Check that core.Core satisfies ClientCore.
 	_   ClientCore = (*core.Core)(nil)
-	log slog.Logger
+	log dex.Logger
 	// errUnknownCmd is wrapped when the command is not know.
 	errUnknownCmd = errors.New("unknown command")
 )
 
 // ClientCore is satisfied by core.Core.
 type ClientCore interface {
+	websocket.Core
```

Comment:
It looks like the only method from `websocket.Core` being used is `WalletState`, and [that one is going away soon](https://github.com/decred/dcrdex/pull/585#discussion_r466701535), it seems. I don't think that embedding the `websocket.Core` is worth it.

I also noticed that **webserver.clientCore** is not exported, but **rpcserver.ClientCore** and **websocket.Core** are exported. These are only used internally for testing purposes, no?

Reply:
You`re right. My initial thinking was that this core will also need to satisfy the one for websocket, and while it does, it doesn't need to be declared here.